<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="urdf_try_1">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="0.115060400545944 -0.115447922845765 -0.0025"
        rpy="0 0 0" />
      <mass
        value="0.205048064278336" />
      <inertia
        ixx="0.000744636102725019"
        ixy="-3.13646252611694E-07"
        ixz="-9.04560815030523E-23"
        iyy="0.000837402621952737"
        iyz="2.53540352214626E-21"
        izz="0.00158118435774326" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="lf_bracket_link">
    <inertial>
      <origin
        xyz="-0.02 0.025986763677896 -0.00990787914229635"
        rpy="0 0 0" />
      <mass
        value="0.00833672226393641" />
      <inertia
        ixx="2.98092057068024E-06"
        ixy="-7.41612608992847E-22"
        ixz="4.30983900644287E-22"
        iyy="2.5814430021137E-06"
        iyz="8.42085226413098E-07"
        izz="2.70784033899702E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lf_bracket_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lf_bracket_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="lf_bracket_join"
    type="fixed">
    <origin
      xyz="0.2025 -0.08625 -0.005"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="lf_bracket_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="lf_DCmotor_link">
    <inertial>
      <origin
        xyz="-0.0035 0.031837 -0.01"
        rpy="0 0 0" />
      <mass
        value="0.064538" />
      <inertia
        ixx="2.7927E-05"
        ixy="-2.3139E-21"
        ixz="2.4025E-22"
        iyy="1.0166E-05"
        iyz="2.865E-21"
        izz="2.7916E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lf_DCmotor_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lf_DCmotor_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="lf_DCmotor_join"
    type="fixed">
    <origin
      xyz="-0.0165 -0.0256 -0.0146"
      rpy="0 0 0" />
    <parent
      link="lf_bracket_link" />
    <child
      link="lf_DCmotor_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="lf_wheel_link">
    <inertial>
      <origin
        xyz="1.11022302462516E-16 -0.0149570892186662 0"
        rpy="0 0 0" />
      <mass
        value="0.131781957836458" />
      <inertia
        ixx="5.64750546860615E-05"
        ixy="-5.55353366772088E-21"
        ixz="-1.68196837805002E-21"
        iyy="9.31833010061579E-05"
        iyz="2.03038431756179E-21"
        izz="5.64750546860616E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lf_wheel_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lf_wheel_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="lf_wheel_join"
    type="continuous">
    <origin
      xyz="-0.0035 0.0992 -0.01"
      rpy="0 0 0" />
    <parent
      link="lf_DCmotor_link" />
    <child
      link="lf_wheel_link" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="lb_bracket_link">
    <inertial>
      <origin
        xyz="0.02 0.025986763677896 -0.00990787914229634"
        rpy="0 0 0" />
      <mass
        value="0.00833672226393639" />
      <inertia
        ixx="2.98092057068024E-06"
        ixy="-4.55595242922701E-22"
        ixz="2.49362026391E-22"
        iyy="2.5814430021137E-06"
        iyz="8.42085226413098E-07"
        izz="2.70784033899702E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lb_bracket_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lb_bracket_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="lb_bracket_join"
    type="fixed">
    <origin
      xyz="0.0275 -0.08625 -0.005"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="lb_bracket_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="lb_DCmotor_link">
    <inertial>
      <origin
        xyz="0.0035 0.031837 -0.01"
        rpy="0 0 0" />
      <mass
        value="0.064538" />
      <inertia
        ixx="2.7927E-05"
        ixy="-1.1842E-21"
        ixz="1.8111E-22"
        iyy="1.0166E-05"
        iyz="2.8924E-21"
        izz="2.7916E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lb_DCmotor_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lb_DCmotor_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="lb_DCmotor_join"
    type="fixed">
    <origin
      xyz="0.0165000000000002 -0.0256 -0.0146"
      rpy="0 0 0" />
    <parent
      link="lb_bracket_link" />
    <child
      link="lb_DCmotor_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="lb_wheel_link">
    <inertial>
      <origin
        xyz="-2.77555756156289E-17 -0.0149570892186649 0"
        rpy="0 0 0" />
      <mass
        value="0.131781957836458" />
      <inertia
        ixx="5.64750546860615E-05"
        ixy="2.39861413140954E-22"
        ixz="-8.34935430795748E-22"
        iyy="9.31833010061579E-05"
        iyz="2.4554189164289E-21"
        izz="5.64750546860616E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lb_wheel_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.792156862745098 0.819607843137255 0.933333333333333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/lb_wheel_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="lb_wheel_join"
    type="continuous">
    <origin
      xyz="0.0035 0.0992 -0.01"
      rpy="0 0 0" />
    <parent
      link="lb_DCmotor_link" />
    <child
      link="lb_wheel_link" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="rf_bracket_link">
    <inertial>
      <origin
        xyz="-0.02 -0.025987 -0.0099079"
        rpy="0 0 0" />
      <mass
        value="0.0083367" />
      <inertia
        ixx="2.9809E-06"
        ixy="2.7262E-21"
        ixz="3.5803E-21"
        iyy="2.5814E-06"
        iyz="-8.4209E-07"
        izz="2.7078E-06" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rf_bracket_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rf_bracket_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="rf_bracket_join"
    type="fixed">
    <origin
      xyz="0.2025 -0.14375 -0.00499999999999998"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="rf_bracket_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="rf_DCmotor_link">
    <inertial>
      <origin
        xyz="-0.0035 -0.031837 -0.01"
        rpy="0 0 0" />
      <mass
        value="0.064538" />
      <inertia
        ixx="2.7927E-05"
        ixy="-2.3576E-21"
        ixz="-2.4525E-22"
        iyy="1.0166E-05"
        iyz="-7.2302E-22"
        izz="2.7916E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rf_DCmotor_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rf_DCmotor_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="rf_DCmotor_join"
    type="fixed">
    <origin
      xyz="-0.0165000000000001 0.0256 -0.0146"
      rpy="0 0 0" />
    <parent
      link="rf_bracket_link" />
    <child
      link="rf_DCmotor_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="rf_wheel_link">
    <inertial>
      <origin
        xyz="5.5511E-17 0.014957 0"
        rpy="0 0 0" />
      <mass
        value="0.13178" />
      <inertia
        ixx="5.6475E-05"
        ixy="1.4555E-21"
        ixz="6.8695E-21"
        iyy="9.3183E-05"
        iyz="-1.2048E-21"
        izz="5.6475E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rf_wheel_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rf_wheel_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="rf_wheel_join"
    type="continuous">
    <origin
      xyz="-0.0035 -0.0992 -0.01"
      rpy="0 0 0" />
    <parent
      link="rf_DCmotor_link" />
    <child
      link="rf_wheel_link" />
    <axis
      xyz="0 -1 0" />
  </joint>
  <link
    name="rb_bracket_join">
    <inertial>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <mass
        value="0" />
      <inertia
        ixx="0"
        ixy="0"
        ixz="0"
        iyy="0"
        iyz="0"
        izz="0" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rb_bracket_join.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rb_bracket_join.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="rb_bracket_link"
    type="fixed">
    <origin
      xyz="0.0274999999999996 -0.14375 -0.00499999999999999"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="rb_bracket_join" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="rb_DCmotor_link">
    <inertial>
      <origin
        xyz="0.0035 -0.031837 -0.01"
        rpy="0 0 0" />
      <mass
        value="0.064538" />
      <inertia
        ixx="2.7927E-05"
        ixy="-2.3574E-21"
        ixz="-2.4528E-22"
        iyy="1.0166E-05"
        iyz="-7.0849E-22"
        izz="2.7916E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rb_DCmotor_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rb_DCmotor_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="rb_DCmotor_join"
    type="fixed">
    <origin
      xyz="0.0165 0.0256 -0.0146"
      rpy="0 0 0" />
    <parent
      link="rb_bracket_join" />
    <child
      link="rb_DCmotor_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="rb_wheel_link">
    <inertial>
      <origin
        xyz="0 0.014957 0"
        rpy="0 0 0" />
      <mass
        value="0.13178" />
      <inertia
        ixx="5.6475E-05"
        ixy="1.4555E-21"
        ixz="6.8695E-21"
        iyy="9.3183E-05"
        iyz="-1.1518E-21"
        izz="5.6475E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rb_wheel_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://urdf_try_1/meshes/rb_wheel_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="rb_wheel_join"
    type="continuous">
    <origin
      xyz="0.0035 -0.0992 -0.01"
      rpy="0 0 0" />
    <parent
      link="rb_DCmotor_link" />
    <child
      link="rb_wheel_link" />
    <axis
      xyz="0 -1 0" />
  </joint>
</robot>